inst/WORDLIST

```diff
@@ -1,27 +1,10 @@
-ADSL
-ADTTE
-CDISC
-cloneable
-coercible
-dataname
-datanames
-DDL
-dropdown
-FilterState
-flexbox
 Forkers
-funder
 Github
```

Comment:
How about this guy? Can we change it into GitHub?

Reply:
Totally! My obvious oversight! Fixed https://github.com/insightsengineering/teal.slice/pull/331/commits/b675c07cd36175624741586cfc337fd0cc7ec1e2